content/v2.0/users/create-user.md

```diff
@@ -24,26 +24,30 @@ Use the [`influx user create` command](/v2.0/reference/cli/influx/user/create)
 to create a new user. A new user requires the following:
 
 - A username
+- The organization name or organization ID to add the user to _(provided in the output of
+  [`influx org find`](/v2.0/reference/cli/influx/org/find/))_
 
 ```sh
 # Pattern
-influx user create -n <username>
+influx user create -n <username> -o <org-name>
 
 # Example
-influx user create -n johndoe
+influx user create -n johndoe -o example-org
 ```
 
 ### Create a user with a password and organization
 To create a new user with a password and add the user as a member of an organization,
```

Comment:
To create a new user, use the `influx...` command, and include the following:
- Username
- Org....